docs/regions.md

```diff
@@ -0,0 +1,58 @@
+## NetherGames Regions
+
+::: info PLEASE NOTE
+We've changed na.nethergames.org to us.nethergames.org as of 18 May 2024
+:::
+
+At NetherGames, we pride ourselves with different regions for our players to play our server in. We are willing to provide a list of regions from different locations world wide to connect to. While there might not be a region in your country to you, but you can try connecting to the nearest region. Below are some regions we support as of 18 May 2024:
+
+### Default
+We've a default location connector for players as well, which redirects you to the nearest data-center for connecting, i.e., if you are closer to US, you will be connected to Beauharnois, Canada region, if you are closer to IND, you will be connected to central Mumbai, India region and so on. Here are some steps to connect to it:
+
+1. Open Minecraft
+2. In servers tab, create a new server
+3. In IP field, enter the IP address: play.nethergames.org
+4. (Optional) Set a name for the server, for instance, we'll be keeping it as "NetherGames"
+5. Hit Play, if you've entered all the details carefully, you should be able to connect to the server.
+
+### IND
+On #FILL, we added an opportunity for players from places near to India or India itself, to connect to our server with a better latency and bandwidth speed. These servers were being hosted in central Mumbai, India as of 18 May 2024. Here is how you can connect to it:
```

Comment:
Our IND servers are still hosted in Mumbai, and we are unable to change this anytime soon due to our datacentre provider (OVH) not having any closer locations to central India.

Reply:
alright.